src/PrestaShopBundle/Resources/config/routing/admin/improve/international/country.yml

```diff
@@ -0,0 +1,16 @@
+#todo uncomment all legacy links when page is finished.
```

Comment:
```suggestion
```

Reply:
Added this todo to make sure we dont forget to remove it later on when page is finished. Now we dont want to override the controller because its not finished.

Reply:
We usually know we have to uncomment those because it's what actually mark the page as released But an extra comment for clarity is not blocking here